docs/experiments-20newsgroups.md

```diff
@@ -41,34 +41,57 @@ Now you should see the train and test splits merged into one folder in `collecti
 To index train and test together:
 
 ```bash
-sh target/appassembler/bin/IndexCollection -collection TwentyNewsgroupsCollection \
- -input collections/20newsgroups/20news-bydate \
- -index indexes/lucene-index.20newsgroups.all \
- -generator DefaultLuceneDocumentGenerator -threads 2 \
- -storePositions -storeDocvectors -storeRaw -optimize
+java -cp target/anserini-*-fatjar.jar io.anserini.index.IndexCollection \
```

Comment:
Let's change to `bin/run.sh`?

Reply:
Good point, just replaced them. Should be good now